parser_test.go

```diff
@@ -110,12 +261,20 @@ func TestParse(t *testing.T) {
 			continue
 		}
 
-		tags, err := Parse(testCase.filename, testCase.relative, basepath)
+		var extra FieldSet
+		if testCase.withExtraSymbols {
+			extra = FieldSet{Receiver: true, Module: true}
+		}
```

Comment:
This block can be replaced with
```go
extra := FieldSet{Receiver: testCase.withExtraSymbols}
```

Reply:
I can not do this change, the variable is used outside of the scope of the if body

Reply:
I actually meant replacing the entire if-block, but this can be cleaned up later.

Reply:
I read too fast your comment and miss the point